### PR TITLE
docs: kotlin core api reference

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

docs: kotlin core api reference
We can navigate between different API docs.

## Linear Issue Reference

* LOG-2279

## Testing

image
image
image

### CODE REVIEW

File: packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md
Comment (demonzoo) [inline packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:6]: The file name seems weird. Is this on purpose?

Comment (xiaoyijun) [reply packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:6]: The `-a` represents `A` and the `a` represents `a` because the URI is not case sensitive here.
This file is auto-generated by kotlin.

**Review by IceHe — APPROVED** [review]
LGTM. Better wait for another approval

File: packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md
Comment (demonzoo) [reply packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:6]: ok

**Review by gao-sun — CHANGES_REQUESTED** [review]
no this one is ugly, mind to find a better solution or align some styles?

File: packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md
Comment (simeng-li) [reply packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:6]: Are those auto-generated by a third-party library? Looks so weird.

Comment (xiaoyijun) [reply packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:4]: Yes

**Review by gao-sun — APPROVED** [review]
cool to me, please update pr description image before merging

File: packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md
Comment (xiaoyijun) [reply packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:4]: Please refer to this PR: https://github.com/Kotlin/dokka/issues/2229
Maybe someday we can provide a PR for kotlin/dokka. 🤓
@simeng-li

Comment (simeng-li) [reply packages/docs/docs/sdk/Kotlin/kotlin/io.logto.sdk.core.constant/-claim-name/-a-t_-h-a-s-h.md:4]: exactly what I am feeling, should be configurable.

**Pull request merged**
